receiver/prometheusreceiver/metrics_receiver_labels_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"testing"
 
 	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/stretchr/testify/require"
```

Comment:
How is it that a dependency on require is being added here but uses of it are being removed?  Something is not right here.

Reply:
Hi Anthony this is coming from Merge. I didnt need `require.Greater(t, len(rms), 0, "At least one resource metric should be present")` as `verifyNumScrapeResults(t, td, rms)` is doing the same thing in a better way. This `require` is coming from merge with `verifyLabelLimitTarget1` test, as it is used there. I will co-ordinate and comeback if this is not needed and update accordingly.

Reply:
@Aneurysm9 thanks, Updated. Removed  `require.Greater(t, len(rms), 0, "At least one resource metric should be present")` from `verifyLabelLimitTarget1`